intuita-webview/src/sourceControl/CreatePRView/index.tsx

```diff
@@ -100,32 +102,40 @@ const CreatePR = ({
 						))}
 					</VSCodeDropdown>
 				</div>
-				<VSCodeTextField
-					placeholder="title"
-					value={title}
-					onInput={(e) =>
-						setFormData({
-							...formData,
-							title: (e.target as HTMLInputElement).value,
-						})
-					}
-				>
-					Title
-				</VSCodeTextField>
-				<VSCodeTextArea
-					placeholder="Description"
-					value={body}
-					onInput={(e) =>
-						setFormData({
-							...formData,
-							body: (e.target as HTMLInputElement).value,
-						})
-					}
-				>
-					Body
-				</VSCodeTextArea>
+				{!PRAlreadyExists ? (
+					<>
+						<VSCodeTextField
+							placeholder="title"
+							value={title}
+							onInput={(e) =>
```

Comment:
can you put this function as a named function in the body? The level of indentation here is pretty high.